templates/ai-ml.html

```diff
@@ -1,374 +1,419 @@
-            </div>
-            <p>スクリプトをルートとして実行します</p>
-            <div class="p-code-snippet">
-              <pre class="p-code-snippet__block--icon"><code>sudo ./install-kubeflow.sh</code></pre>
-            </div>
+          <div class="col-6 col-medium-2">
+            <p class="u-text--muted">NVIDIAプラットフォームアライアンス担当ディレクター、</p>
           </div>
-        </li>
-      </ol>
-    </div>
-  </div>
-</section>
-<section class="p-strip--light is-bordered">
-  <div class="row">
-    <div class="col-12">
-      <h3>利用開始へ向けたコンサルティングと、業務に集中するためのマネージドオペレーション</h3>
-      <p>機械学習を幅広く理解するためにワークショップを活用していください。開発者ステーションからデータセンター、パブリッククラウドへの完全なパイプラインを構築します。Canonicalは大手企業との協力によりお客様に幅広い選択肢を提供します。まず当社の標準ベアメタルKubernetesサービスパッケージ（DiscovererまたはDiscoverer Plus）から開始し、次にAIアドオンを選択することでKubernetesのAIを有効に活用できます。</p>
-      <p><a href="/kubernetes">CanonicalのKubernetesパッケージに関する詳細&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
     </div>
-  </div>
+  </section>
 
-  <div class="row">
-    <div class="p-card">
-      <div class="col-12 p-card__header">
-        <h2 class="p-heading--4">AI/MLアドオン</h2>
-        <p><span class="p-heading--2">40,000ドル</span><br>専門のAI/MLワークショップおよび準備状況評価</p>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>モジュール型プラットフォームによりエッジや大型クラウドでAIを実行</h2>
       </div>
-      <div class="row">
-        <div class="col-6">
-          <p>TensorflowやJupyterHubを含むKubeflowに特化したオプションの1日ワークショップです。オンプレミス/オフプレミスでのAI/MLの運用に必要なすべての内容が含まれています。</p>
-          <ul class="p-list">
-            <li class="p-list__item is-ticked">オンサイトまたはリモートのオプション</li>
-            <li class="p-list__item is-ticked">K8sとKubeflowのハンズオン</li>
-            <li class="p-list__item is-ticked">フルパイプラインビュー</li>
-          </ul>
-        </div>
-        <div class="col-6">
-          <p>Canonicalとデータサイエンス分野のパートナーがAI評価をワークショップの一部として提供し、MLやAIのアプローチに対してお客様の既存のデータと分析の準備状況評価をお手伝いします。</p>
-          <ul class="p-list">
-            <li class="p-list__item is-ticked">AIのライフサイクルの理解</li>
-            <li class="p-list__item is-ticked">予備データおよびプロセスのディスカバリ</li>
-            <li class="p-list__item is-ticked">開発能力の評価</li>
-            <li class="p-list__item is-ticked">ML分析のデプロイと運用</li>
-            <li class="p-list__item is-ticked">初期AI戦略の完了</li>
-          </ul>
+      <div class="col">
+        <div class="p-image-container u-hide--small">
+          {{ image(url="https://assets.ubuntu.com/v1/7fb505b4-Canonical%20AI%20ML%20Illustrations%20v5-08%201.jpg",
+                    alt="",
+                    width="600",
+                    height="300",
+                    hi_def=True,
+                    attrs={"class": "p-image-container__image"},
+                    loading="lazy") | safe
+          }}
         </div>
+        <p>実運用グレードのプロジェクトには、拡張性、再現性、移植性を備えたソリューションが必要です。CanonicalのMLOpsは、AIプロジェクトのタイムラインを短縮します：</p>
+        <hr class="is-muted" />
+        <ul class="p-list--divided u-no-margin--bottom">
```

Comment:
I don't think this is needed but feel free to confirm with design
```suggestion
        <ul class="p-list--divided">
```